api/contrib/envoy/extensions/filters/network/postgres_proxy/v3alpha/postgres_proxy.proto

```diff
@@ -20,6 +20,21 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#extension: envoy.filters.network.postgres_proxy]
 
 message PostgresProxy {
+  // Upstream SSL operational modes.
+  enum SSLMode {
+    // Do not encrypt upstream connection to the server.
+    SSL_DISABLE = 0;
```

Comment:
no need SSL_ prefix for them. Just use `DISABLED`, `REQUIRED`.

Reply:
Done.